Pandas.py

```diff
@@ -0,0 +1,43 @@
+import pandas as pd
+import numpy as np
+
+# function to read gff files
+def read_gff(gfffile):
+    df = pd.read_csv(gfffile, sep='\t', comment='t', header = 0, names=['Chromosome', 'source',
+                                                                        'type', 'start',
+                                                                        'end', 'score',
+                                                                        'strand', 'phase', 'atributes'])
```

Comment:
Твой код учитывает только случай, где комментарий занимает только первую строку. Однако чаще всего в файлах данного формата могут быть комментарии на много строк, их нужно учитывать. Благо в пандасе это делается супер просто
```suggestion
    df = pd.read_csv(gfffile, sep='\t', comment="#", names=['Chromosome', 'source',
                                                                        'type', 'start',
                                                                        'end', 'score',
                                                                        'strand', 'phase', 'atributes'])
```

Reply:
Ещё я не совсем понял, что значит `comment="t"`, в чём виновата эта бедная буква(((

Reply:
А, я понял, кажется, но тогда это костыль)